osquery/sql/tests/sql_tests.cpp

```diff
@@ -154,4 +154,11 @@ TEST_F(SQLTests, test_sql_sha256) {
   EXPECT_EQ(d[0]["test"],
             "9f86d081884c7d659a2feaa0c55ad015a3bf4f1b2b0b822cd15d6c15b0f00a08");
 }
+
+TEST_F(SQLTests, test_sql_regex_replace) {
```

Comment:
Tests failed but in a bizarre (not so much) way.
I try to use those parameters : 
regex_replace('un éléphant', '[é]', '-')
result was 'un --l--phant' instead of 'un -l-phant
No crash

regex_replace('/cage/éléphant/', '(éléphant|oiseau)', '-')
result was '/cage/-/' as expected

So our anonymization is not compromise